std/sys/io/Process.hx

```diff
@@ -22,10 +22,20 @@
 package sys.io;
 
 extern class Process {
+	/**
+		Standard output. The output stream where a process writes its output data.
+	**/
+	var stdout(default, null) : haxe.io.Input;
+
+	/**
+		Standard error. The output stream to output error messages or diagnostics.
```

Comment:
This requires at least a "typically", but I'm not quite sure if we should add these layman-one-liners at all.